src/common/types/feed.js

```diff
@@ -1,5 +1,10 @@
 // @flow strict
 import type { MessageDescriptor } from 'react-intl';
+import type {
+    AnnotationFileVersion,
+    AnnotationReply,
+    Target,
+} from '../../elements/content-sidebar/activity-feed/annotations/types';
```

Comment:
Can we make it so that the types are declared in the common file and imported by the component, instead?

Reply:
Sure, I was just separating concerns a bit but I can put them in this file